src/features/rejection/rejection-sagas.spec.js

```diff
@@ -0,0 +1,62 @@
+import { describe } from 'riteway';
+import { call, put, select } from 'redux-saga/effects';
+
+import reducer, { createQuestion, loadQuestions, getState } from './rejection-reducer';
+import { loadLocalState, saveLocalState } from './rejection-sagas';
+import { loadState, saveState } from '../../localStorage';
+
+describe('rejection sagas', async assert => {
+  {
+
+    const gen = loadLocalState();
+
+    assert({
+      given: 'the loadLocalState saga',
+      should: 'first call the loadState function',
+      actual: gen.next().value,
+      expected: call(loadState)
+    });
+
+    assert({
+      given: 'the loadLocalState saga',
+      should: 'then put the loadQuestions function with the loaded state',
+      actual: gen.next().value,
```

Comment:
You don't need undefined if you pass an expected value into `gen.next(expectedValue)`

Reply:
Updated to input test state.